### ISSUE TITLE
[Groupees] not retaining login info

### ISSUE THREAD

**Issue by Brekkik** [Jeshibu/PlayniteExtensions#40]
Any time the Playnite client restarts, it will pops up 2 error notifications regarding the Groupees plugin:

"Failed to import games from Groupees.
Delegate to an instance method cannot have null 'this'."

and

"Error fetching Groupees games: User not authenticated"

**Comment by Brekkik** [comment]
[extensions.log](https://github.com/Jeshibu/PlayniteExtensions/files/12528935/extensions.log)
[playnite.log](https://github.com/Jeshibu/PlayniteExtensions/files/12528937/playnite.log)

**Comment by Jeshibu** [comment]
Thanks! I'm only seeing the "User no authenticated" one there. You're 100% sure `main menu > Add-ons > Extension settings > Libraries > Groupees` said you were authenticated at that exact moment? The current 2FA is confusing the login flow a bit, but if you navigate to your profile page afterwards, it should work.

**Comment by Brekkik** [comment]
Right, I forgot to mention when I posted the logs that I haven't been able to get the null error to show up again since I started trying to log it, so I'm not sure what I did, but I guess it's fixed at this point. If that changes, I'll be sure to make a new report on it.

As for the authentication error, I do make sure it says I'm logged in each time. I log in via Steam though and it doesn't ask me to do any 2FA after I click the button to log in with Steam. I just click it, and then the window closes and my Authenticate status on my Libraries extension settings says "Logged in," but it has to be logged in again every time I restart it.
